packages/zoe/test/unitTests/contracts/test-simpleExchange.js

```diff
@@ -163,9 +163,9 @@ test('simpleExchange with valid offers', async t => {
   // Alice had 3 moola and 0 simoleans.
   // Bob had 0 moola and 7 simoleans.
   t.equals(aliceMoolaPurse.getCurrentAmount().extent, 0);
-  t.equals(aliceSimoleanPurse.getCurrentAmount().extent, 7);
+  t.equals(aliceSimoleanPurse.getCurrentAmount().extent, 4);
```

Comment:
Interesting. Which change had an effect on behavior?

Reply:
This was the change in how swap works, where the excess is left in the original offer.